rero_ils/modules/documents/api_views.py

```
@@ -24,21 +24,29 @@
 
 
 @api_blueprint.route("/cover/<isbn>")
-@cached(timeout=5 * 60, query_string=True)  # 5 minutes timeout
 def cover(isbn):
     """Document cover service."""
-    return jsonify(get_remote_cover(isbn))
+
+    use_cache = flask_request.args.get("cached", default="true").lower() != "false"
+    try:
+        url, provider = get_thumbnail_url(isbn, cached=use_cache)
+    except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)